CRM/MembersOnlyEvent/Upgrader.php

```diff
@@ -1,7 +1,21 @@
 <?php
+use CRM_MembersOnlyEvent_ExtensionUtil as E;
 
 /**
- * Collection of upgrade steps
+ * Collection of upgrade steps.
  */
 class CRM_MembersOnlyEvent_Upgrader extends CRM_MembersOnlyEvent_Upgrader_Base {
+
+  /**
+   * Adds non_member_price_field_value table to DB.
+   * Adds entity_price_field_value table to DB.
+   */
+  public function upgrade_0001() {
```

Comment:
Why use 0001 instead of 2000? As this step is for version 2.0?

Reply:
I thought it should be increased incrementally and I've checked `membershipextras`